src/bp/ui-studio/src/web/views/FlowBuilder/nodeProps/SaySomethingForm/index.tsx

```diff
@@ -0,0 +1,266 @@
+import { Button as BPButton, Checkbox, Position, Toaster } from '@blueprintjs/core'
+import { MoreOptions, MoreOptionsItems, style as sharedStyle } from 'botpress/shared'
+import classnames from 'classnames'
+import _ from 'lodash'
+import React, { FC, Fragment, useEffect, useReducer, useState } from 'react'
+
+import withLanguage from '../../../../components/Util/withLanguage'
+import { getFormData, isFormEmpty } from '../../../../util/NodeFormData'
+import EditableInput from '../../common/EditableInput'
+import style from '../style.scss'
+
+import SaySomethingTextForm from './TextForm'
+
+const { MoreOptionsStyles } = sharedStyle
+
+interface Props {
+  buffer: any
+  categories: any
+  contentLang: string
+  defaultLanguage: string
+  copyFlowNode: any
+  onDeleteSelectedElements: () => void
+  fetchContentCategories: any
+  flow: any
+  node: any
+  pasteFlowNode: any
+  readOnly: boolean
+  requestEditSkill: any
+  subflows: any
+  updateFlow: any
+  updateNode: any
+  user: any
```

Comment:
Please separate props as OwnProps, DispatchProps and StateProps, refer to https://github.com/botpress/botpress/blob/minor/12-8-0/src/bp/ui-studio/src/web/views/OneFlow/index.tsx to use existing typings for redux props